### DISCUSSION TITLE
"Failed to install Camoufox on Mac OS"

### DISCUSSION THREAD

**Discussion by yangxiaowei111** [daijro/camoufox#102] [General]
1  pip install -U camoufox[geoip]  successed
2 python -m camoufox fetch  failed
Camoufox successfully installed.
Downloading GeoIP database: 100%|█████████| 58.4M/58.4M [00:12<00:00, 4.69MiB/s]
Failed to download and extract UBO: 451 Client Error: unknown for url: https://addons.mozilla.org/firefox/downloads/file/4382536/ublock_origin-1.61.0.xpi

from camoufox.sync_api import Camoufox

with Camoufox() as browser:
    page = browser.new_page()
    
raise InvalidAddonPath(
camoufox.exceptions.InvalidAddonPath: manifest.json is missing. Addon path must be a path to an extracted addon.

**Comment by daijro** [comment]
Hello,

Are you living in a country where access to mozilla.org is blocked? Looks like the URL was blocked for legal reasons.

I will add uBlock Origin's GitHub release download URL as a backup.
